pod/perlfunc.pod

```diff
@@ -4326,6 +4326,8 @@ a restrictive MODE and give the user no way to be more permissive.
 The exceptions to this rule are when the file or directory should be
 kept private (mail files, for instance).  The documentation for
 L<C<umask>|/umask EXPR> discusses the choice of MODE in more detail.
+If bits in MODE other than the permission bits are set, the result may
+be implementation defined, per POSIX 1003.1-2008.
```

Comment:
Would it be desirable and possible to supply a URL to the location in POSIX where this is discussed?

Reply:
It's here: https://pubs.opengroup.org/onlinepubs/9699919799/functions/mkdir.html

I'm not sure on desirable.

Reply:
Okay.  Thanks for the link.